docs/publishers/cli.md

```diff
@@ -0,0 +1,65 @@
+# DPM: the datapackage manger CLI
+
+The dpm is a command-line tool aimed to help publishers to prepare and upload data to the Datapackage Registry. With dpm you will be able to:
+
+* Validate your data to ensure its quality
+* Upload datapackage
+* Tag uploaded datapackage to create historical snapshot
+* Remove uploaded datapackage that is no longer needed
+
+## Installation
+
+Dpm is available on pypi [hopefully soon it will be], you can install it using pip in the terminal:
```

Comment:
I'd suggest `Dpm` is always `dpm` lower case as it is a command line tool not a class or object.